### ISSUE TITLE
Switch dates to an unambiguous format

### ISSUE THREAD

**Issue by rossant** [cortex-lab/alyx#446]
e.g. 05-Sept-2018 to avoid confusion with EU/US format
suggested by Anne Urai

**Comment by nsteinme** [comment]
The current date format is unambiguous because no one ever uses yyyy-dd-mm so when you see yyyy-??-?? it always has to be yyyy-mm-dd. This is a global standard, ISO8601, I recommend we keep it. [https://xkcd.com/1179/](https://xkcd.com/1179/)

**Comment by rossant** [comment]
@nsteinme sure, the standard for storing dates should remain unchanged. But for display and user input we should avoid `09/10/2018`

cc @anne-urai

**Comment by nsteinme** [comment]
It is not already? Here's what I see: 
![image](https://user-images.githubusercontent.com/7590975/45644029-771bf280-bab4-11e8-993f-94281ab1d49c.png)

**Comment by rossant** [comment]
yes but click on a subject and you should see dd/mm/yyyy in the input widget

**Comment by anne-urai** [comment]
<img width="512" alt="screen shot 2018-09-17 at 15 05 33" src="https://user-images.githubusercontent.com/6636245/45644218-29d75b00-ba8b-11e8-8edd-3da1e633bfd0.png">

When adding a new subject the date of birth entry box uses dd/mm/yyyy. I agree that either everything should me mmmm-dd-yyyy or yyyy-mm-dd (probably the latter when reading in dates from the data into python/matlab, since it's a bit easier to parse and work with?)

**Comment by nsteinme** [comment]
I see what you're referring to now. Yes, let's go with the latter (switch those input boxes to yyyy-mm-dd), I agree.
